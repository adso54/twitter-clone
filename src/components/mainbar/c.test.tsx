import { render } from '@testing-library/react';
import '@testing-library/jest-dom'
import MainBar from './index'

test('Rendered MainBar match the snapshot', async () => {
    const component = render(MainBar())
    expect(component).toMatchSnapshot()
})