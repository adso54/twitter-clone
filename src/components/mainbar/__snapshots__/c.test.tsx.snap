// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendered MainBar match the snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bczRLJ gYKYKU"
      >
        <div
          class="sc-gsnTZi fOaQIp"
        >
          <div
            class="sc-eCYdqJ huTjpC"
          >
            <div
              class="sc-gKXOVf uFCtg"
            >
              <svg>
                user.svg
              </svg>
            </div>
            <div
              class="sc-jSMfEi dehjIL"
            >
              Profile
            </div>
          </div>
        </div>
        <div
          class="sc-dkzDqf zkpvH"
        >
          Główna
          <div
            class="sc-hKMtZM bKkhMB"
          >
            <svg>
              star.svg
            </svg>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bczRLJ gYKYKU"
    >
      <div
        class="sc-gsnTZi fOaQIp"
      >
        <div
          class="sc-eCYdqJ huTjpC"
        >
          <div
            class="sc-gKXOVf uFCtg"
          >
            <svg>
              user.svg
            </svg>
          </div>
          <div
            class="sc-jSMfEi dehjIL"
          >
            Profile
          </div>
        </div>
      </div>
      <div
        class="sc-dkzDqf zkpvH"
      >
        Główna
        <div
          class="sc-hKMtZM bKkhMB"
        >
          <svg>
            star.svg
          </svg>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
